stdlib/public/Concurrency/Task+init.swift.gyb

```diff
@@ -253,12 +253,13 @@ extension Task where Failure == ${FAILURE_TYPE} {
 % end
   ///
 % if IS_DETACHED:
-  /// Don't use a detached unstructured task if it's possible
-  /// to model the operation using structured concurrency features like child tasks.
```

Comment:
My understanding was that this guidance still stands -- what's the reason to stop recommending against detached tasks?

Reply:
You're right to point this out. In 824d47b613c I have reintroduced this guidance, but in such a way that it is no longer limited to the `IS_DETACHED` condition. This guidance is equally applicable to all unstructured tasks, so with my changes in 824d47b613c it should appear in the documentation unconditionally (it has been phrased so as not to mention "detached" specifically).